src/devices/tuya.ts

```diff
@@ -9672,6 +9672,13 @@ export const definitions: DefinitionWithExtend[] = [
             e.produced_energy().withDescription('Total reverse active energy'),
             e.power_factor().withUnit('%'),
             e.ac_frequency(),
+            e
+                .numeric('data_report_duration', ea.SET)
```

Comment:
```suggestion
                .numeric('data_report_interval', ea.SET)
```

Would this name make more sense?

Reply:
I am ok either way, the reason for the name I used is that in Tuya app that is how it is called when updating firmware - [see screenshot here](https://github.com/Koenkk/zigbee2mqtt/issues/22236#issuecomment-2325492263).

Reply:
Then let's keep it like this.